tests/unit/db/account_tests.py

```diff
@@ -450,6 +450,31 @@ def test_billing_data(self):
         finally:
             self.client.disconnect()
 
+    def test_set_year_without_month_for_billing_data(self):
+        """
+        Test raising an exception when retrieving billing data with only
+        year parameter
+        """
+        try:
+            self.client.connect()
+            year = 2016
+            self.assertRaises(CloudantException, self.client.bill, year)
```

Comment:
Change this to use the assertRaises context manager and then to assert that the error that is returned is the desired message.  For example:

```python
with self.assertRaises(CloudantArgumentError) as cm:
    self.client.bill(year)
expected = ('Invalid year and/or month supplied.  Found: year - 2016, month - None')
self.assertEqual(str(cm.exception), expected)
```

This also works into our plans to update all of our exception testing to use this pattern.  See #80.